index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <h1 id="title"> freeCodeCamp Survey Form </h1>
+    <p id="description"> Thank you for taking the time to help us improve the platform </p>
+    <form id="survey-form">
+      <div>
+        <label id = "name-label" for = "name">Name</label>
+        <input id="name" required type="text" placeholder=" Enter your name">
```

Comment:
* Hi, @Lockless404 this is an example of how to use inline comments 
* Click on the File changed the link and select the block of code where you want to request a change on and comment below the code ( ie see how i commented under the code on line 18, i can now make my review saying " Kindly use classes instead of id's, this is the best practice as it enables you to use the styling in multiple places  ")

Reply:
Thank you.